### PR TITLE
Feature/72 mapping patient in icu

### PR DESCRIPTION
**Pull request by jsachau**
Finished Patient in ICU Mapping. This fixes issue [#72](https://github.com/ehrbase/num_platform/issues/72).

### CODE REVIEW

**Comment by ppazos** [comment]
@jsachau  can you please check the conflicts with develop branch?

**Comment by jsachau** [comment]
@ppazos I have merged the newest develop branch

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 1]
```diff
@@ -278,15 +278,8 @@ public void createObservationUsingDefaultProfile() {
...
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Default profile is not supported for Observation. One of the following profiles is expected: [http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, http://hl7.org/fhir/StructureDefinition/heartrate, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/patient-in-icu, https://charite.infectioncontrol.de/fhir/core/StructureDefinition/CoronavirusNachweisTest, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/pregnancy-status, https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:281]: @jsachau can you please use the multiline layout? it is easier to read.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 2]
```diff
@@ -299,15 +292,8 @@ public void createObservationUsingUnsupportedProfile() {
...
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Profile http://hl7.org/fhir/StructureDefinition/vitalsigns is not supported for Observation. One of the following profiles is expected: [http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, http://hl7.org/fhir/StructureDefinition/heartrate, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/patient-in-icu, https://charite.infectioncontrol.de/fhir/core/StructureDefinition/CoronavirusNachweisTest, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/pregnancy-status, https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:295]: @jsachau can you please use the multiline layout? it is easier to read.

**Review by ppazos — CHANGES_REQUESTED** [review]

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java (hunk 1 above)
Comment (jsachau) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java]: @ppazos yes i did.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java (hunk 2 above)
Comment (jsachau) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java]: @ppazos there too. Sorry!

**Comment by jsachau** [comment]
@ppazos I have updated the code a few days ago. The requested change is outdated since then.

**Review by ppazos — APPROVED** [review]

**Pull request merged**